.travis.yml

```diff
@@ -5,8 +5,8 @@ before_install:
 script:
         - ./make.sh
         - make check
-        - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cp libcapstone.so bindings/python/; fi
-        - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cp libcapstone.dylib bindings/python/; fi
+        - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cp libcapstone.so.* bindings/python/libcapstone.so; fi
```

Comment:
it’s ok then :)

> On 03 May 2016, at 15:57, Pranith Kumar <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/aquynh/capstone/pull/661#discussion_r61884814>:
> 
> > @@ -5,8 +5,8 @@ before_install:
> >  script:
> >          - ./make.sh
> >          - make check
> > -        - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cp libcapstone.so bindings/python/; fi
> > -        - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cp libcapstone.dylib bindings/python/; fi
> > +        - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cp libcapstone.so.* bindings/python/libcapstone.so; fi
> This is the travis test script. The test VM is erased after every run, so there is no chance of recompilation.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/aquynh/capstone/pull/661/files/fbf1cd7822ee9f7ad0c16b9f6458a95f9b0ca5d5..97aecf4bf98a3ec5932e928e9a74ab4d62d2c306#r61884814>